src/app/dashboard/dashboard.component.ts

```diff
@@ -92,13 +93,20 @@ export class DashboardComponent implements OnInit {
       && shelf.resourceIds.length === 0;
   }
 
-  getSurveys() {
-    this.submissionsService.getSubmissions(findDocuments({
-      'user.name': this.userService.get().name,
-      type: 'survey',
-       status: 'pending'
-    })).subscribe((surveys) => {
-      this.surveys = surveys;
+  getSurveysSubmissions() {
+    forkJoin([

```

Comment:
There's no need to put these requests together in one function or `forkJoin` observable.  You should split this up into two functions which will also allow for clearer variable and function names.